docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -18,12 +18,13 @@ A formula should be deprecated to indicate to users that the formula should not
 
 The most common reasons for deprecation are when the upstream project is deprecated, unmaintained, or archived.
 
-Formulae with dependents may be deprecated only if at least one of the following are true:
+Formulae should only be be deprecated if at least one of the following are true:
 
-- its dependents are all deprecated
 - the formula does not build on any of our supported macOS versions and on Linux
 - the formula has outstanding CVEs
 
+Formulae with dependents should not be deprecated until or when all dependents are also deprecated.
```

Comment:
Isn't this a bit redundant with the previous block?

Reply:
It makes sense to me as written. Though I think `should` should be changed to `cannot` since we literally won't let you do such a thing with our audits.

Reply:
```suggestion
Formulae with dependents cannot not be deprecated until or when all dependents are also deprecated.
```